openedx/core/djangoapps/user_api/views.py

```diff
@@ -669,26 +669,58 @@ def _add_terms_of_service_field(self, form_desc, required=True):
         terms_text = _(u"Terms of Service")
         terms_link = u"<a href=\"{url}\">{terms_text}</a>".format(
             url=marketing_link("TOS"),
-            terms_text=terms_text
+            terms_text=terms_text,
         )
 
-        # Translators: "Terms of service" is a legal document users must agree to
```

Comment:
@stvstnfrd @caesar2164 There is a paragraph comment in opened/djangoapps/user_api/views.py  line 287 discussing TOS, honor code, and privacy policy check boxes...supporting flexibility..etc.  I know the logistration has been updated in dogwood, so our change may never make it upstream.

This may have been up